packages/pds/src/db/util.ts

```diff
@@ -1,22 +1,15 @@
 import {
   DummyDriver,
   DynamicModule,
+  Kysely,
```

Comment:
Now might be a good opportunity to update to the latest kysely.

Reply:
oohhh i'd really like to do that

a bit worried that the api mighta changed a decent bit tho. i can try upgrading on another branch & checkout how much work it'd be

Reply:
Yeah, no biggie if it's not the right time!